docs/1000.yaml

```
@@ -1325,9 +1331,9 @@ Q1153441:
 
 Q1153584:
   title: Monotone convergence theorem
-  # TODO: many versions in mathlib, choose one
-  # e.g. integral_tendsto_of_tendsto_of_antitone (Bochner integral)
-  # or lintegral_tendsto_of_tendsto_of_monotone (Lebesgue integral)
+  decls:
+    - integral_tendsto_of_tendsto_of_antitone
+    - lintegral_tendsto_of_tendsto_of_monotone
```

I think this should fix the build error, hopefully:
```suggestion
    - Measure_theory.integral_tendsto_of_tendsto_of_antitone
    - Measure_theory.lintegral_tendsto_of_tendsto_of_monotone
```

Let's try and get this in to fix the webpage again:
bors d+